tests/components/lcn/test_helpers.py

```diff
@@ -0,0 +1,31 @@
+"""Test for helper functions."""
```

Comment:
We should avoid testing (or interacting with) integration internals directly, as noted in https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations

Are we able to test these without interacting with this module directly?

Reply:
Yes, that's possible. I removed the unit test and tested the failing `get_resource` method within the webcomponents module.